[2023-12-21T21:21:08.208+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pizza_order_dag.check_order_task 2023-12-21T21:21:07.581Z [queued]>
[2023-12-21T21:21:08.216+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pizza_order_dag.check_order_task 2023-12-21T21:21:07.581Z [queued]>
[2023-12-21T21:21:08.217+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-21T21:21:08.230+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): check_order_task> on 2023-12-21 21:21:07.581000+00:00
[2023-12-21T21:21:08.234+0000] {standard_task_runner.py:57} INFO - Started process 620 to run task
[2023-12-21T21:21:08.236+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'pizza_order_dag', 'check_order_task', '2023-12-21T21:21:07.581Z', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/pizza_order_dag.py', '--cfg-path', '/tmp/tmptgy9d9oj']
[2023-12-21T21:21:08.237+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask check_order_task
[2023-12-21T21:21:08.275+0000] {task_command.py:416} INFO - Running <TaskInstance: pizza_order_dag.check_order_task 2023-12-21T21:21:07.581Z [running]> on host 20e184a2eb37
[2023-12-21T21:21:08.345+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pizza_order_dag' AIRFLOW_CTX_TASK_ID='check_order_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-21T21:21:07.581000+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='2023-12-21T21:21:07.581Z'
[2023-12-21T21:21:08.546+0000] {logging_mixin.py:154} INFO - [X] - all right
[2023-12-21T21:21:08.547+0000] {python.py:194} INFO - Done. Returned value was: 1
[2023-12-21T21:21:08.725+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=pizza_order_dag, task_id=check_order_task, execution_date=20231221T212107, start_date=20231221T212108, end_date=20231221T212108
[2023-12-21T21:21:08.771+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-21T21:21:08.788+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
